### PR TITLE
Converted nativeAuth TTL to string date format and changed UI description.

### PR DESCRIPTION
**Pull request by tdumitrache**
### Issue/Feature
Converted nativeAuth TTL to string date format and changed UI description.

### Reproduce
Issue exists on version `2.20.0` of sdk-dapp.

### Contains breaking changes
[x] No
[] Yes

### Updated CHANGELOG
[x] Yes

### Testing
[x] User testing
[x] Unit tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- [Converted nativeAuth TTL to string date format and changed UI description](https://github.com/multiversx/mx-sdk-dapp/pull/910)
```
Comment (arhtudormorar) [inline CHANGELOG.md:10]: Changed Ledger authentication texts for v.23 of MultiversX app

File: src/UI/ledger/LedgerLoginContainer/ConfirmAddress/ConfirmAddress.tsx [hunk 2]
```diff
@@ -68,7 +74,7 @@ export const ConfirmAddress = ({
             ledgerModalConfirmDescriptionClassName
           )}
         >
-          For security, please confirm that your address:
```
Comment (arhtudormorar) [inline src/UI/ledger/LedgerLoginContainer/ConfirmAddress/ConfirmAddress.tsx:71]: for ledger v22 all remains unchanged

File: src/UI/ledger/LedgerLoginContainer/ConfirmAddress/ConfirmAddress.tsx [hunk 3]
```diff
@@ -89,7 +95,7 @@ export const ConfirmAddress = ({
               ledgerModalConfirmDescriptionClassName
             )}
           >
-            and Auth Token:
+            and authorize:
```
Comment (arhtudormorar) [inline src/UI/ledger/LedgerLoginContainer/ConfirmAddress/ConfirmAddress.tsx:98]: unchanged for v <23

File: src/UI/ledger/LedgerLoginContainer/ConfirmAddress/helpers/secondsToTimeString.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
+/*
+   Converts number of seconds to a time string. Example: 123 -> "2min 3 sec.";
```
Comment (arhtudormorar) [inline src/UI/ledger/LedgerLoginContainer/ConfirmAddress/helpers/secondsToTimeString.ts:2]: remove comment

File: src/UI/ledger/LedgerLoginContainer/ConfirmAddress/helpers/getAuthTokenText.ts [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  const description = 'are the one shown on your Ledger device screen now.';
+
+  if (ledgerWithUsernames) {
+    const time = secondsToTimeString(nativeAuthInfo.ttl);
+
+    return {
+      data: `${nativeAuthInfo.origin} for ${time}.`,
+      description
```
Comment (arhtudormorar) [inline src/UI/ledger/LedgerLoginContainer/ConfirmAddress/helpers/getAuthTokenText.ts:29]: this is not used outside

File: src/UI/ledger/LedgerLoginContainer/ConfirmAddress/helpers/tests/secondsToTimeString.test.ts [hunk 6]
```diff
@@ -0,0 +1,30 @@
+import { secondsToTimeString } from '../secondsToTimeString';
+
+describe('secondsToTimeString tests', () => {
+  it('converts 0 seconds to "0 sec"', () => {
+    expect(secondsToTimeString(0)).toBe('N/A time');
+  });
+
+  it('converts seconds less than a minute correctly', () => {
```
Comment (arhtudormorar) [inline src/UI/ledger/LedgerLoginContainer/ConfirmAddress/helpers/tests/secondsToTimeString.test.ts:8]: add tests for invalid numbers: (NaN, undefined, -1, null, string)

File: src/UI/ledger/LedgerLoginContainer/ConfirmAddress/helpers/getAuthTokenText.ts (hunk 5 above)
Comment (tdumitrache) [reply src/UI/ledger/LedgerLoginContainer/ConfirmAddress/helpers/getAuthTokenText.ts:29]: It is now

File: src/UI/ledger/LedgerLoginContainer/ConfirmAddress/ConfirmAddress.tsx [hunk 7]
```diff
@@ -41,6 +43,15 @@ export const ConfirmAddress = ({
   const tokenLogin = useSelector(tokenLoginSelector);
   const loginToken = tokenLogin?.loginToken ?? token;
 
+  const authTokenText = getAuthTokenText({
+    loginToken,
+    version: ledgerAccount?.version
+  });
+
+  const { ledgerWithUsernames } = getLedgerVersionOptions(
+    ledgerAccount?.version ?? '0'
```
[the remark this answers is not in the document]
Comment (tdumitrache) [reply src/UI/ledger/LedgerLoginContainer/ConfirmAddress/ConfirmAddress.tsx:52]: Because ledgerAccount: LedgerAccountType | null

File: src/UI/ledger/LedgerLoginContainer/ConfirmAddress/helpers/getAuthTokenText.ts [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+    return null;
+  }
+
+  const { ledgerWithUsernames } = getLedgerVersionOptions(version);
+  const nativeAuthInfo = decodeLoginToken(loginToken);
+  if (nativeAuthInfo == null) {
+    return null;
+  }
+
+  const infoPartOne = 'For security, please confirm that your address:';
```
Comment (arhtudormorar) [inline src/UI/ledger/LedgerLoginContainer/ConfirmAddress/helpers/getAuthTokenText.ts:22]: confirmAddressText

File: src/UI/ledger/LedgerLoginContainer/ConfirmAddress/helpers/getAuthTokenText.ts [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+  const infoPartTwo = 'and Auth Token:';
```
Comment (arhtudormorar) [inline src/UI/ledger/LedgerLoginContainer/ConfirmAddress/helpers/getAuthTokenText.ts:23]: authText

File: src/UI/ledger/LedgerLoginContainer/ConfirmAddress/helpers/getAuthTokenText.ts [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+  const infoPartThree = 'are the one shown on your Ledger device screen now.';
```
Comment (arhtudormorar) [inline src/UI/ledger/LedgerLoginContainer/ConfirmAddress/helpers/getAuthTokenText.ts:24]: areShownText

**Review by arhtudormorar — APPROVED** [review]
